### DISCUSSION TITLE
Azure Service Bus secrets are not picked up in secret scanning

### DISCUSSION THREAD

**Discussion by vermegi** [community/community#33491] [Code Security]
When using Azure Service Bus, the secrets used to connect to a service bus are not picked up by GH secret scanning. Can I report this issue somewhere? 

Secret looks like this: SharedAccessKey=i0lvQYWgAmzvMQOGUSDhT8xYiFjcX17p/GUK+r264mc= 
(key has been rotated)

**Comment by 15MariamS** [comment]
Hi @vermegi, 

Thank you for the feedback! Azure Service Bus secrets are not yet [supported](https://docs.github.com/en/enterprise-cloud@latest/code-security/secret-scanning/secret-scanning-patterns), but I've passed along your feedback to our partners!
